src/main/kotlin/graphics/scenery/primitives/ParticleGlyphs.kt

```diff
@@ -28,20 +36,17 @@ class ParticleGlyphs @JvmOverloads constructor(var positions: List<Vector3f>, va
         }
 
         geometry{
-            vertices.position(0)
-            vertices.limit(positions.size * 3)
-            positions.forEach { v ->
-                v.get(vertices)
-                vertices.position(vertices.position() + 3)
-            }
+
+            vertices = BufferUtils.allocateFloat(positions.limit())
+            vertices.put(positions)
```

Comment:
This is allocating native memory, so your class needs a `close()` method to free that memory again 👍

Reply:
Where do I need to call close then?